### PR TITLE
Fix: APP-760 Review Proposal - Voting details

### PR DESCRIPTION
**Pull request by Fabricevladimir**
## Description
- wires up proposal details to voting terminal on Proposal Review screen

Task: [APP-760](https://aragonassociation.atlassian.net/browse/APP-760)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

### CODE REVIEW

File: packages/web-app/src/containers/votingTerminal/index.tsx [hunk 1]
```diff
@@ -80,6 +57,9 @@ export const VotingTerminal: React.FC<VotingTerminalProps> = ({
   const [selectedVote, setSelectedVote] = useState('');
   const {t} = useTranslation();
 
+  const isTokenBased = token?.symbol !== undefined;
+  console.log('isTokenBased', isTokenBased);
```
Comment (WissenIstNacht) [inline packages/web-app/src/containers/votingTerminal/index.tsx]: ?

File: packages/web-app/src/containers/reviewProposal/index.tsx [hunk 2]
```diff
@@ -33,16 +58,86 @@ const ReviewProposal: React.FC = () => {
     ],
   });
 
+  const startDate = useMemo(
+    () =>
+      `${format(
+        Date.parse(
+          `${values.startDate} ${values.startTime}:00 ${values.startUtc}`
+        ),
+        KNOWN_FORMATS.proposals
```
Comment (WissenIstNacht) [inline packages/web-app/src/containers/reviewProposal/index.tsx]: When I reach the review page I get: `Uncaught RangeError: Invalid time value`. Chrome is fine, it only happens on Firefox.

**Review by WissenIstNacht — CHANGES_REQUESTED** [review]
Thanks man. Unfortunately the browser gods are not on your side today...

File: packages/web-app/src/utils/date.ts [hunk 3]
```diff
@@ -1,8 +1,9 @@
 import {i18n} from '../../i18n.config';
 import {format, formatRelative, formatDistance} from 'date-fns';
 
-const KNOWN_FORMATS = {
+export const KNOWN_FORMATS = {
   standard: 'MMM dd yyyy HH:mm', // This is our standard used for showing dates.
+  proposals: 'yyyy/MM/dd hh:mm a',
```
Comment (sepehr2github) [inline packages/web-app/src/utils/date.ts:6]: I would suggest to follow a same date pattern for proposal setting date input and the date we show on review proposal page

**Review by sepehr2github — APPROVED** [review]
Thanks Fabrice, Once you addressed Vali's Comments. You're good to go!

File: packages/web-app/src/utils/date.ts (hunk 3 above)
Comment (Fabricevladimir) [reply packages/web-app/src/utils/date.ts:6]: I know what you mean. Designs, designs

File: packages/web-app/src/containers/reviewProposal/index.tsx (hunk 2 above)
Comment (Fabricevladimir) [reply packages/web-app/src/containers/reviewProposal/index.tsx]: Actually, this is quite easy to resolve: don't use Firefox 😆 Jk. Thanks for catching that.

**Review by WissenIstNacht — APPROVED** [review]
Not sure what it is you did, but it worked! 🚀

**Pull request merged**